packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/FlatTreeNode.ts

```diff
@@ -49,3 +59,38 @@ function getFlatNodes(nodes: PresentationTreeNode[], level: number) {
   });
   return flatNodes;
 }
+
+export function getErrors(rootNodes: PresentationTreeNode[]) {
+  return rootNodes.flatMap((rootNode) => getErrorNodes(rootNode, []));
+}
+
+function getErrorNodes(parentNode: PresentationTreeNode, path: string[]) {
+  const errorList: ErrorType[] = [];
+  const newPath = [...path];
+
+  if (!isPresentationHierarchyNode(parentNode)) {
```

Comment:
Looks like this code path is can only be hit if one of the root nodes is error node?

Reply:
This has been adjusted